### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by tanruiquan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ Format: `add n/NAME e/EMAIL m/MODULE_CODE [LESSON_CODE(S)] [p/PHONE] [h/TELEGRAM
 * Name, email and module code are **compulsory**
 * Lesson code, phone number, telegram handle and remarks are **optional**
 * You can add multiple module codes, and can have multiple lesson codes for a single module code
+* You will not be able to add a person with the same email/phone number/telegram handle
```
Comment (James-Kua) [inline docs/UserGuide.md:171]:
```suggestion
* You will not be able to add a person with the same email/phone number/telegram handle as another existing contact in contHACKS
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -321,6 +324,7 @@ Format: `editc INDEX [m/MODULE_CODE LESSON_CODE] [d/DAY] [t/START_TIME END_TIME]
 
 * At least one of the parameters must be provided.
 * Existing values of the parameters specified will be erased and updated to the input values
+* You will not be able to edit a lesson into a given module and lesson code if it already exists in contHACKS
```
Comment (James-Kua) [inline docs/UserGuide.md:329]:
```suggestion
* Edited lesson should not contain the same module and lesson code as another existing lesson in contHACKS
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -193,6 +194,7 @@ Format: `edit INDEX [n/NAME] [e/EMAIL] [m/MODULE_CODE LESSON_CODE(S)] [p/PHONE]
 * At least one of the parameters must be provided
 * Existing values of the parameters specified will be erased and updated to the input values
 * You can input multiple module codes, and can have multiple lesson codes for a single module code
+* You will not be able to edit a person into a given email/phone number/telegram handle if it already exists in contHACKS
```
Comment (James-Kua) [inline docs/UserGuide.md:198]:
```suggestion
* Edited person should not contain the same email/phone number/telegram handle as another existing contact in contHACKS
```

**Review by James-Kua — APPROVED** [review]
Some minor suggestions to make it clearer, LGTM!

File: docs/UserGuide.md (hunk 3 above)
Comment (tanruiquan) [reply docs/UserGuide.md:198]: for this i prefer my current version

File: docs/UserGuide.md [hunk 4]
```diff
@@ -298,6 +302,7 @@ Format: `addc m/MODULE_CODE LESSON_CODE d/DAY t/START_TIME END_TIME [r/REMARK]`
 * All the parameters are **compulsory** except for remark
 * Day input is only accepted in integer form <br> e.g. `1` for Monday, `7` for Sunday
 * Start and end time input is only accepted in the `HH:mm` format <br> e.g. `09:00`
+* You will not be able to add a lesson with the same module and lesson code
```
Comment (James-Kua) [inline docs/UserGuide.md:305]:
```suggestion
* You will not be able to add a lesson with the same module and lesson code as another existing lesson in contHACKS
```

**Review by James-Kua — APPROVED** [review]
Sorry, just one more change

**Pull request merged**